src/pages/posts/[slug].js

```diff
@@ -80,6 +80,25 @@ export default function Post({ post }) {
       <Section className={styles.postFooter}>
         <Container>
           <p className={styles.postModified}>Last updated on {formatDate(modified)}.</p>
+          {!!relatedPosts.length && (
+            <div className={styles.relatedPosts}>
+              <span>
+                More from{' '}
+                <Link href={categoryPathBySlug(categories[0].slug)}>
+                  <a>{categories[0].name}</a>
```

Comment:
trying to think about how `relatedPosts` is created, do you think it might make sense to have an object passed into the component that represents both an array of posts and the title? so that you're not inferring the association? thinking in terms of using the functionality elsewhere

for instance, this can be as simple as when passing in `relatedPosts` in `getStaticProps`, something like:
```
relatedPosts: {
  posts: getRelatedPosts({ category: post.categories[0].slug }), // based on earlier thoughts
  title: post.categories[0].name
}
```

thoughts?

Reply:
That is a great idea, making the changes!

Reply:
@colbyfayock  Just one doubt about this: the first version has a link to the category (in the title), so is required to pass at `getStaticProps` at least the slug of the category or the link, something like this: 

```
relatedPosts: {
  posts: ... ,
  title: {
     name: post.categories[0].name
     link:  categoryPathBySlug(categories[0].slug)
   }
}
```
Do you think is a good idea to include this link, or will be somehow redundant because the link is already at Metadata?

Reply:
hm thats a good question. id vote for the link option like the block you posted, simply because it provides someone the ability to change it to something else if they'd like 🤷  maybe someone decides to use a different pattern for linking or something, or wants to add tracking to those links, etc etc idk.